src/manifest.ts

```diff
@@ -41,9 +43,67 @@ export class SlackManifest {
             def.name,
         },
       },
-      "outgoing_domains": def.outgoingDomains || [],
+      settings: {},
     } as ManifestSchema;
 
+    if (def.slackHosted == false) {
```

Comment:
what happens when `def.slackHosted` === undefined?

Reply:
maybe instead of using `def.slackHosted` here, we should use `manifest.function_runtime` as it handles the undefined usecase (sets it to true by default)

Reply:
Using `manifest.function_runtime`  is breaking the union type to be ` ISlackManifestRemote` under its current scope. So to handle` def.slackHosted === undefined` issue I changed it to `(def.slackHosted === false && def.slackHosted !== undefined)`